pkg/squashfs/verity_test.go

```diff
@@ -11,6 +11,47 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type uidmapTestcase struct {
+	uidmap   string
+	expected bool
+}
+
+var uidmapTests = []uidmapTestcase{
+	uidmapTestcase{
+		uidmap:   `         0          0 4294967295`,
+		expected: true,
+	},
+	uidmapTestcase{
+		uidmap: `         0          0 1000
+2000 2000 1`,
+		expected: false,
+	},
+	uidmapTestcase{
+		uidmap:   `         0          0 1000`,
+		expected: false,
+	},
+	uidmapTestcase{
+		uidmap:   `         10          0 4294967295`,
+		expected: false,
+	},
+	uidmapTestcase{
+		uidmap:   `         0          10 4294967295`,
+		expected: false,
+	},
+	uidmapTestcase{
+		uidmap:   `         0          0 1`,
```

Comment:
this is the case that I tested above (I think) and the result should be true.

Reply:
iiuc we're now agreed that this is not the case.